backend/tests/test_logout.py

```diff
@@ -18,3 +31,37 @@ def test_logout_user_without_token(client):
     response = rv.get_json()
     assert rv.status_code == HTTPStatus.UNAUTHORIZED
     assert response['msg'] == 'Missing Authorization Header'
+
+
+def test_logout_twice(app, client, access_token):
+    """Test logout twice."""
+    with app.app_context():
+        rv = client.delete('/auth/logout/',
+                           headers={'Authorization': 'Bearer {}'
+                                    .format(access_token)})
+        assert rv.status_code == HTTPStatus.OK
+        rv = client.delete('/auth/logout/',
+                           headers={'Authorization': 'Bearer {}'
+                                    .format(access_token)})
+        tokens = JWTToken.query.all()
+    response = rv.get_json()
+    assert len(tokens) == 1
+    assert tokens[0].revoked
+    assert rv.status_code == HTTPStatus.UNAUTHORIZED
+    assert response["msg"] == "token has been revoked"
```

Comment:
ditto

Reply:
It is behavior implemented in endpoint (not related to the test). Biside naming convencion for messages, we should not checking if token was revoked or not manually in every endpoint, as it is now (flask_jwt_extended has method to do it autmomatically). I added #208 targeting that problem.